src/pages/Home/context/provider.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import reducer from './uni-verse-reducer';
+import initialState from './uni-verse-initial-state';
+import { Context } from './uni-verse-context';
```

Comment:
What was wrong with global-context ? :D the naming is bad

Reply:
There will be two independent contexts one for Tak Beyt UI which is old model (Uni-Verse) and one for Sher or Chand Beyt UI which is based on GPT-2. I named this on multi-Verse.